### PR TITLE
Finalize templates

### PR DESCRIPTION
**Pull request by anissa-agahchen**
# Overview

* Finalize template configurations
* All moose SRB and Composition rules are collapsed into one set
* can upload a file of any name

### CODE REVIEW

File: api/src/utils/media/csv/validation/csv-row-validator.ts [hunk 1]
```diff
@@ -320,6 +330,10 @@ export const getValidFormatFieldsValidator = (config?: ColumnFormatValidatorConf
       }
 
       const rowValueForColumn = String(row[columnIndex]);
+
+      if (rowValueForColumn === 'undefined') {
```
Comment (sdevalapurkar) [inline api/src/utils/media/csv/validation/csv-row-validator.ts]: we have to check for the value equaling the actual string `undefined`?

File: database/src/migrations/20210909000000_validation_schemas.ts [hunk 2]
```diff
@@ -31,82 +25,27 @@ enum SPECIES_NAME {
...
-    species: SPECIES_NAME.MOOSE,
-    template: TEMPLATE_NAME.MOOSE_SRB_OR_COMPOSITION_SURVEY_SKEENA
-  },
-  {
-    fileName: csm1_wld4147_t2.toString(),
-    cms: COMMON_SURVEY_METHODOLOGY.STRATIFIED_RANDOM_BLOCK,
-    species: SPECIES_NAME.MOOSE,
-    template: TEMPLATE_NAME.MOOSE_SRB_OR_COMPOSITION_SURVEY_OMINECA
-  },
+  //Common Survey Methodology: Stratified Random Block or Composition
```
Comment (sdevalapurkar) [inline database/src/migrations/20210909000000_validation_schemas.ts]:
```suggestion
  // Common Survey Methodology: Stratified Random Block or Composition
```

File: api/src/utils/media/csv/validation/csv-row-validator.ts (hunk 1 above)
Comment (anissa-agahchen) [reply api/src/utils/media/csv/validation/csv-row-validator.ts]: fixed it ... good catch.

File: app/src/features/observations/components/ObservationSubmissionCSV.tsx [hunk 3]
```diff
@@ -109,7 +109,7 @@ const ObservationSubmissionCSV: React.FC<IObservationSubmissionCSVProps> = (prop
 
   return (
     <>
-      <Tabs value={value} onChange={handleChange} aria-label="csv-groups" indicatorColor="primary">
+      <Tabs value={value} onChange={handleChange} aria-label="csv-groups" indicatorColor="primary" variant="scrollable">
```
Comment (sdevalapurkar) [inline app/src/features/observations/components/ObservationSubmissionCSV.tsx:112]: nice!

**Review by sdevalapurkar — DISMISSED** [review]
lgtm

**Review by sdevalapurkar — APPROVED** [review]

**Pull request merged**